AirCasting/APICommunicator/APICommunication.swift

```diff
@@ -0,0 +1,31 @@
+// Created by Lunar on 01/07/2021.
+//
+
+import Foundation
+
+final class APICommunication {
+    func forgotPassword(login: String, completion: @escaping (Int) -> Void) {
+        let params = ["user": ["login": login]]
+
+        var request = URLRequest(url: URL(string: "http://aircasting.org/users/password.json")!)
+        request.httpMethod = "POST"
+        request.httpBody = try? JSONSerialization.data(withJSONObject: params, options: [])
+        request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+        let session = URLSession.shared
```

Comment:
we're not using `URLSession` directly as we've the `APIClient` protocol to abstract away networking calls. Please use `APIClient` instead.